azblob/zc_uuid.go

```diff
@@ -21,10 +21,7 @@ type uuid [16]byte
 func newUUID() (u uuid) {
 	u = uuid{}
 	// Set all bits to randomly (or pseudo-randomly) chosen values.
-	_, err := rand.Read(u[:])
-	if err != nil {
-		panic("ran.Read failed")
-	}
+	rand.Read(u[:])
```

Comment:
Are we ok with the case read fails?

Reply:
Yes, this error practically never occurs. If we didn't want to panic on it, then we should simply ignore it, since the user wouldn't know how to deal with it even if we propagate it upwards.